doc/lsst.pipe.base/command-line-task-retargeting-howto.rst

```diff
@@ -0,0 +1,132 @@
+.. _command-line-task-retargeting-howto:
+
+##########################################
+Retargeting subtasks of command-line tasks
+##########################################
+
+Subtasks of :ref:`command-line tasks <using-command-line-tasks>` are dynamically retargetable, meaning that you can configure what task class is run by a parent class.
+Subtask retargeting is a special case of :ref:`command-line task configuration <command-line-task-config-howto>`.
+
+A common use of retargeting is to change a default subtask for a camera-specific one.
+For example:
+
+- `lsst.obs.subaru.isr.SuprimeCamIsrTask`: a version of instrument signature removal (ISR or detrending) for Suprime-Cam and Hyper Suprime-Cam.
+- `lsst.obs.sdss.selectSdssImages.SelectSdssImagesTask`: an version of the task that selects images for co-addition of SDSS stripe 82 images.
+
+How you retarget a subtask and override its config parameters depends on whether the task is specified as an `lsst.pex.config.ConfigurableField` (the most common case) or as an `lsst.pex.config.RegistryField`.
+Both scenarios are described on this page.
+
```

Comment:
Please describe why you'd use one and when the other, what the tradeoffs are, etc.